ui/death_knight/frost/presets.ts

```diff
@@ -71,17 +72,17 @@ export const P1_MASTERFROST_EP_PRESET = PresetUtils.makePresetEpWeights(
 			[Stat.StatStrength]: 2.86,
 			[Stat.StatArmor]: 0.03,
 			[Stat.StatAttackPower]: 1,
-			[Stat.StatExpertise]: 0.75,
-			[Stat.StatMeleeHaste]: 1.38,
-			[Stat.StatMeleeHit]: 1.08,
-			[Stat.StatMeleeCrit]: 0.64,
-			[Stat.StatSpellHit]: 0.59,
-			[Stat.StatSpellCrit]: 0.43,
-			[Stat.StatMastery]: 1.41,
+			[Stat.StatExpertiseRating]: 0.75,
+			[Stat.StatHasteRating]: 1.38,
+			[Stat.StatHitRating]: (1.08 + 0.59),
```

Comment:
I assume the next time we calculate stats where Melee/Spell are added together we just add the combined value from the Stat Weight modal?

Reply:
This should be done automatically by the back-end before the weight values are sent back to the UI, assuming I implemented it correctly:
https://github.com/wowsims/cata/pull/883/commits/5bf44821257225b11a9d8dcfaaade13f29f7fe11